### PR TITLE
[FIX] sale_project: put recurrent services only in project subs section

### PR DESCRIPTION
**Pull request by pivi-odoo**
## Current behaviour
When selling a recurrent service with fixed/prepaid invoicing, it appears two times in the project profitability report, once as "Fixed/Prepaid" Service, another time as a "Subscription".

## Expected behaviour
Such service should only be present in the "Subscription" section.

## Steps to reproduce
- Install Sales, Project, Subscription
- Create a Service that is invoiced "Fixed/Prepaid", creates a new project on creation
- Create a new SO with that product, confirm it.
- Go to the Project's Update of the newly created project
- Observe that the services was accounted 2 times, onces in the "Fixed/Prepaid" section, another time in the "Subscription".

## Reason for the problem
When calculating the profitability from "Fixed/Prepaid" services, we ignore the fact if they are a subscription or not.

## Fix
Extend the domain to not take subscription into account when generating the profitability items that are prepaid services.

## Affected versions
- 16.0
- master
---
opw-3113320

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by nle-odoo — APPROVED** [review]
looks good to me, equivalent code but this allows override in project_sale_subscription for https://github.com/odoo/enterprise/pull/35765

File: addons/sale_project/models/project.py [hunk 1]
```diff
@@ -403,6 +403,16 @@ def _get_revenues_items_from_sol(self, domain=None, with_action=True):
             'total': {'to_invoice': total_to_invoice, 'invoiced': total_invoiced},
         }
 
+    def _get_revenues_items_from_invoices_domain(self, domain=None):
+        if domain is None:
+            domain = []
+        return expression.AND([
+            domain,
+            [('move_id.move_type', 'in', ['out_invoice', 'out_refund']),
```
Comment (xavierbol) [inline addons/sale_project/models/project.py:411]: (Thanks to one of your last fixes) we could use `get_sale_types` method. 🙂 
```suggestion
            [('move_id.move_type', 'in', self.env['account.move'].get_sale_types()),
```

What do you think? 🙂

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I left one comment. 🙂

**Review by xavierbol — APPROVED** [review]
LGTM, thanks again for the fix. 🙂 

robodoo r+

**Pull request closed without merging**